packages/cdk8s/BUILD.bazel

```diff
@@ -0,0 +1,104 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//:packages.bzl", "CONSTRUCTS_PACKAGE_VERSION")
+load("//tools:defaults.bzl", "jest_test", "jsii_library", "jsii_package", "package_json", "ts_library")
+
+MODULE_NAME = "cdk8s"
+
+# Note, we could easily make this into a macro
+JSII_TARGETS = str({
+    "dotnet": {
+        "namespace": "Org.Cdk8s",
+        "packageId": "Org.Cdk8s",
+    },
+    "java": {
+        "maven": {
+            "artifactId": "cdk8s",
+            "groupId": "org.cdk8s",
+        },
+        "package": "org.cdk8s",
+    },
+    "python": {
+        "distName": "cdk8s",
+        "module": "cdk8s",
+    },
+})
+
+package_json(
+    name = "package_json",
+    package_name = MODULE_NAME,
+    bundled_deps = ["json-stable-stringify", "yaml"],
+    desc = "Cloud Development Kit for Kubernetes",
+    experimental = True,
+    jsii_targets = JSII_TARGETS,
+    peer_deps = {
+        "constructs": CONSTRUCTS_PACKAGE_VERSION,
```

Comment:
I expected `constructs` to also appear in `devDeps` here. I actually want to make sure the `devDep` is pinned on 2.0.0 because that's the lowest version we currently support.

Reply:
`devDeps` are not needed for package publishing, otherwise package version is handled solely at the root package.json level.